src/features/home/components/VenuesModule.tsx

```diff
@@ -4,21 +4,29 @@ import { FlatList, ListRenderItem } from 'react-native'
 import { ModuleTitle } from 'features/home/atoms'
 import { VenueTile } from 'features/home/atoms/VenueTile'
 import { DisplayParametersFields } from 'features/home/contentful'
+import { GeoCoordinates } from 'libs/geolocation'
+import { formatDistance } from 'libs/parsers'
 import { VenueHit } from 'libs/search'
 import { Spacer } from 'ui/theme'
 
 type VenuesModuleProps = {
   hits: VenueHit[]
   display: DisplayParametersFields
+  position: GeoCoordinates | null
```

Comment:
```suggestion
  userLocation: GeoCoordinates | null
```

pour que ce soit plus explicite

Reply:
Oui je suis d'accord avec toi, ça me perturbais aussi d'avoir 2 fois "position"